packages/css/src/components/viewer-grid/viewer-grid.scss

```diff
@@ -132,6 +132,23 @@ $dso-viewer-grid-overlay-width: 624px;
   overflow-y: auto;
   z-index: 1;
 
+  .dso-filterblok {
+    margin-left: $grid-gutter-width * -0.5;
```

Comment:
Zouden we dit 'naar buiten trekken' middels een mixin moeten doen? Hetzelfde zie ik ook in `.dso-document-header` staan...

Reply:
Kan, maar andere keer